javascript/javascript2/week1/js2w1hw-hyf.js

```diff
@@ -0,0 +1,105 @@
+//Write a function that finds the shortest word of an array of words ex---
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWords(danishWords) {
+let shortValue = danishWords[0];
+   for(const shortWord of danishWords){
+     if(shortValue.length > shortWord.length){
+       shortValue = shortWord;
+     }
+   }
+   return shortValue
+  }
+console.log("Shortest word from string array = "+ shortestWords(danishWords));
+
+
+////Find the individual number and the total number of Danish letters in a string. ex---
+const danishString = "Jeg har en blå bil";
+//const danishString2 = "Blå grød med røde bær";
+const danishString2 = "Bl grød med røde br";
+
```

Comment:
Good work Charmi) In my humble opinion, I would recommend you not to leave "dead comments", just delete it. In this case, it seems to me that line 17 should be uncommented and line 18 just deleted (perhaps).

Reply:
ohh..actually I tried with different sentences n forgot to delete that ..but Thanks dear to review my hw and giving me a Feedback ..I will correct it...